Tweak zero

colinrford opened the pull request:
Here are some minor tweaks I've made since **Sprint 1** in hopes to unify our app a little bit. This is _not_ complete, so please do not merge just yet! This is the time for you all to look at this branch and critique. I would like to have this PR merged by **Tuesday evening**, if possible.

Initially, I was going to change everything from activities over to View Models. We changed our mind on that. Instead, I am working on converting activities to fragments. 

I have also been working on adding the navigation drawer to everyone's module. @brandonschabell and @MrJohnson, we're going to need to work together a little bit on this so that I don't wreck your branches from merging this PR. @avaidyam, @Wall-Dough and @Brahmancer, I do not believe this branch will affect much of what you all are doing, but I could be wrong so please give me a heads up if so. 

This branch also will disable tablet use and disable landscape use in our app, indefinitely. 

In short, _these are small yet crucial tweaks that will allow development for our app to proceed smoothly!_

Commit:
Merge branch 'master' into tweak-zero

First few tweaks

Commit:
Minor tweaks

Need to commit these changes to update from master

Commit:
Merge branch 'master' into tweak-zero

# Conflicts:
#	Health/src/main/AndroidManifest.xml
#	Health/src/main/java/com/sciencesquad/health/MainActivity.java
Resolving conflicts made after the “workouts” branch was merged into
“master”

Commit:
Uniformity attempt

Attempting to clean up a little bit; unify the UI some. Need to test
still.

Commit:
Merge branch 'master' into tweak-zero

# Conflicts:
#	Health/build.gradle

Commit:
Adding NavigationView

Added nav views to alarm and clock modules

Commit:
No Tablets, No Landscape

Self explanatory

avaidyam commented on Health/src/main/AndroidManifest.xml:
```diff
@@ -56,36 +75,46 @@
+            android:screenOrientation="portrait" />
+
+        <!-- For Workout testing. -->
+        <activity
```
These need to be killed.

avaidyam commented on Health/src/main/java/com/sciencesquad/health/ClockActivity.java:
```diff
@@ -1,20 +1,24 @@
-import java.util.Calendar;
-
-public class ClockActivity extends BaseActivity {
+public class ClockActivity extends BaseActivity implements
```
`Fragment`?

colinrford replied:
@avaidyam not yet. this was written pre-fragment decision

avaidyam commented on Health/src/main/java/com/sciencesquad/health/activity/MapsActivity.java:
```diff
@@ -314,4 +323,43 @@ public void onLocationChanged(Location location) {
+     */
+    @SuppressWarnings("StatementWithEmptyBody")
+    @Override
+    public boolean onNavigationItemSelected(MenuItem item) {
```
Please don't do this. :cry: Remove all of these from everything except `MainActivity`.

Commit:
Incremental work

Minor changes since last commit; making this commit so that @avaidyam
and I can hopefully transition from activities to fragments

Commit:
Fragment work; Manifest fix

Fixed AndroidManifest, should be done for now. Doing more stuff with
fragments. Doing it a silly way perhaps. Sry

Commit:
Merge branch 'master' into tweak-zero

# Conflicts:
#	Health/src/main/AndroidManifest.xml
#	Health/src/main/java/com/sciencesquad/health/MainActivity.java
